src/styles/typography/index.md

```diff
@@ -157,6 +157,12 @@ For example, links in a header or side navigation might not need underlines. Use
 
 {{ example({group: "styles", item: "typography", example: "link-no-underline", html: true, open: true}) }}
 
+### Links to change a language
+
+Ensure that links that allow a user to change a language are understandable even when out of context.
```

Comment:
```suggestion
Make sure links that allow a user to change a language are understandable even when out of context.
```

Reply:
OR
When you include a link that allows a user to change languages, make sure its function is always clear, even when taken out of context.

Reply:
How about something with more detail?
```suggestion
Links can be used to allow a user to access the current content in a different language.

When offering links to content in other languages, make sure:
- the link's content includes the language name in both English and the source language
- the link's purpose is always clear, even when taken out of context
- the link element includes an [hreflang attribute](https://www.w3schools.com/tags/att_a_hreflang.asp) that identifies the language of the linked page.
```